client/components/mma/identity/identity.ts

```diff
@@ -113,7 +113,14 @@ export const ConsentOptions: ConsentOptionCollection = {
 		return await RemoveSubscriptionsAPI.execute();
 	},
 	newsletters(options: ConsentOption[]): ConsentOption[] {
-		return options.filter(isNewsletter);
+		return (
+			options
+				.filter(isNewsletter)
+				// @AB_TEST: Default Onboarding Newsletter Test: START
+				// Delete this line when trial is over
+				.filter((newsletter: ConsentOption) => newsletter.id !== 'XXXX') // TODO
```

Comment:
It would be clearer which newsletter this is if we use the `newsletter.identityName` (unique kebab-cased name) for the filter instead of the `newsletter.id`. eg the test could be something like `newsletter.identityName ===  'default-onboarding-test-newsletter'`

We'll be able to get the actual value for the identityName when we add the new newsletter to the list. I'll reach out about doing that today

Reply:
```suggestion
				.filter((newsletter: ConsentOption) => newsletter.identityName !== 'saturday-roundup-trial') 
```

Reply:
I'd prefer to use the numerical id becuase the identityName is an optional field in the code as this is also used for consents.  

```
export interface ConsentOption {
	id: string;
	description?: string;
	frequency?: string;
	name: string;
	theme?: string;
	type: ConsentOptionType;
	subscribed: boolean;
	identityName?: string;
	isProduct?: boolean;
	isChannel?: boolean;
}
```

Reply:
👍  thanks for talking me through that!